specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2023-10-01/mfe.json

```diff
@@ -10086,6 +10110,156 @@
       },
       "additionalProperties": false
     },
+    "CategoricalDataDriftMetric": {
+      "enum": [
+        "JensenShannonDistance",
+        "PopulationStabilityIndex",
+        "PearsonsChiSquaredTest"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "CategoricalDataDriftMetric",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "JensenShannonDistance",
+            "description": "The Jensen Shannon Distance (JSD) metric."
+          },
+          {
+            "value": "PopulationStabilityIndex",
+            "description": "The Population Stability Index (PSI) metric."
+          },
+          {
+            "value": "PearsonsChiSquaredTest",
+            "description": "The Pearsons Chi Squared Test metric."
+          }
+        ]
+      },
+      "additionalProperties": false
```

Comment:
>"additionalProperties": false [](http://example.com/codeflow?start=6&length=29)

The additionalProperties is not required to defined in all properties, if it is false.

Reply:
this seems to be a broader change that'd be required across the whole API definition - For this check-in, I think it'd be best to follow the convention throughout the whole definition and make that change as a separate PR :)